osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs

```diff
@@ -238,7 +238,7 @@ private double computeSpeedValue(ScoreInfo score, OsuDifficultyAttributes attrib
             double relevantAccuracy = attributes.SpeedNoteCount == 0 ? 0 : (relevantCountGreat * 6.0 + relevantCountOk * 2.0 + relevantCountMeh) / (attributes.SpeedNoteCount * 6.0);
 
             // Scale the speed value with accuracy and OD.
-            speedValue *= (0.95 + Math.Pow(attributes.OverallDifficulty, 2) / 750) * Math.Pow((accuracy + relevantAccuracy) / 2.0, (14.5 - attributes.OverallDifficulty) / 2);
+            speedValue *= (0.95 + Math.Pow(Math.Max(0, attributes.OverallDifficulty), 2) / 750) * Math.Pow((accuracy + relevantAccuracy) / 2.0, (14.5 - attributes.OverallDifficulty) / 2);
```

Comment:
```suggestion
            speedValue *= (0.95 + Math.Pow(Math.Max(0, attributes.OverallDifficulty), 2) / 750) * Math.Pow((accuracy + relevantAccuracy) / 2.0, (14.5 - Math.Max(0, attributes.OverallDifficulty)) / 2);
```

Reply:
That actually makes scaling slightly worse (green this pr, purple your change)
![image](https://github.com/user-attachments/assets/40f40f4d-8377-4d87-af3d-a4f33d05f298)

Reply:
Yeah I guess it's fine because it's not raising to a power there.